8.0/apache/buster/debug/entrypoint.sh

```diff
@@ -22,16 +20,16 @@ then
   if [ -z "$HOST" ]; then
       HOST=`/sbin/ip route|awk '/default/ { print $3 }'`
   fi
-  
+
   # xdebug config
   if [ -f /usr/local/etc/php/conf.d/20-xdebug.ini ]
   then
-      sed -i "s/xdebug\.remote_host \=.*/xdebug\.remote_host\=$HOST/g" /usr/local/etc/php/conf.d/20-xdebug.ini
+      sed -i "s/xdebug\.client_host \=.*/xdebug\.client_host\=$HOST/g" /usr/local/etc/php/conf.d/20-xdebug.ini
   fi
 
   if [ -f /etc/php/7.1/cli/conf.d/20-xdebug.ini ]
   then
-      sed -i "s/xdebug\.remote_host \=.*/xdebug\.remote_host\=$HOST/g" /etc/php/7.0/cli/conf.d/20-xdebug.ini
+      sed -i "s/xdebug\.client_host \=.*/xdebug\.client_host\=$HOST/g" /etc/php/7.0/cli/conf.d/20-xdebug.ini
   fi
```

Comment:
I think those broken lines should finally be removed now ;)

Reply:
?

Reply:
This file is for PHP 8 but line 32 checks for an `xdebug.ini` of PHP 7.1 and then manipulates the one for PHP 7.0 ;)

See #19 which already deleted those lines (but only in the global template).

Reply:
fixed, can you check again pls

Reply:
Looks good to me